### ISSUE TITLE
Records from import need correcting

### ISSUE THREAD

**Issue by kitenetter** [BiologicalRecordsCentre/iRecord#966]
User ID 151060 uploaded a spreadsheet of 449 records on 9 December 2020, into the "iRecord import" survey (survey ID 93), resulting in 449 records with occurrence IDs 18363096 to 18363545.

The grid reference was imported as SU099978 but should have been SU099939 for all of the records.

Some of these records have already been verified, so I'm wondering if we can do an update that does not trigger them for re-verification. But if that isn't possible then a correction to the grid reference is in any case needed.

**Comment by kitenetter** [comment]
Ignore my comment about not re-triggering verification. In fact none of the incorrect records have been verified, and the change of grid ref also results in a change of vice-county, so we do need to reindex the records following the grid ref change, and to ensure that they are seen as pending verification.

**Comment by burkmarr** [comment]
@kitenetter - only 412 of those 449 records have a GR of 'SU099978' - the others have a variety of similar GRs (only one of which matches the GR which you asked me to change it to). Shall I just update the 412 or all 449 (448 not including the one already with the correct GR)?

Running this gives 449 records. Uncommenting the GR constraint gives 412:
```sql
select * from samples s 
join occurrences o on o.sample_id = s.id
where s.survey_id = 93 
--and (s.entered_sref='SU099978')
and date(s.created_on) ='2020-12-09'
and s.created_by_id = 151060
and o.id >= 18363096 
and o.id <= 18363545
```

**Comment by kitenetter** [comment]
@burkmarr I've now corrected these manually via the website. The original issue arose because the user was unable to edit this set of records at the time he contacted us, not sure why that was the case. But since they are now editable I decided to get them sorted out that way.

In the process I noticed that the same user has managed to get a whole load of records entered with incrementing grid refs (no doubt thanks to Excel), so there may be another request soon ...

Closing this one.
